src/Controller/Customer.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Customer as CustomerEntity;
+use App\Entity\User as UserEntity;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\Extension\Core\Type\EmailType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+use Doctrine\ORM\EntityRepository;
+
+
+class Customer extends AbstractController
+{
+
+    public function index(Request $request)
+    {
+        $customers = $this->getDoctrine()->getRepository(CustomerEntity::class)->findAll();
+        return $this->render('customers.html.twig', array('customers' => $customers));
+    }
+
+    public function new(Request $request)
+    {
+
+        $customer = new CustomerEntity();
+
+        $form = $this->createFormBuilder($customer)
+            ->add('name', TextType::class, ['label' => "Nome Completo"])
+            ->add('document_number', TextType::class, ['label' => "Número documento"])
+            ->add('email', EmailType::class, ['label' => "E-mail"])
+            ->add('phone', TextType::class, ['label' => "Telefone"])
+            ->add('user',EntityType::class, [
+                'class' => UserEntity::class,
+                'query_builder' => function(EntityRepository $er){
+                    return $er->createQueryBuilder('user')
+                    ->where('user.role = 3');
+                },
+                'choice_label' => 'name',
+                'label' => "Vendedor",
+                'placeholder' => 'Selecione um vendedor',
```

Comment:
We need to add placeholder in the selects to add attribute "required". 

Unless the customer is not required to be from a seller.... :confused: